docs/sources/configure/_index.md

```diff
@@ -1,8 +1,11 @@
 ---
-description: Describes parameters used to configure Grafana Loki.
-menuTitle: Configuration parameters
 title: Grafana Loki configuration parameters
-weight: 500
+menuTitle: Configuration parameters
+description: Configuration reference for the parameters used to configure Grafana Loki.
+aliases: 
+  - ../configuration
+  - ../configure
```

Comment:
I don't think this alias is necessary. I believe it is self referential. Unlike absolute aliases, this won't be correct once the file is moved, so it doesn't make sense to include it for future proofing.

```suggestion
```